fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -180,7 +180,11 @@ def run
         c.description = 'Create a new plugin that can be used with fastlane'
 
         c.action do |args, options|
+<<<<<<< HEAD
           PluginGenerator.new.generate args.shift
+=======
+          PluginGenerator.new.generate
+>>>>>>> 532a9a6fe97ec3038deacb16b2160abcf5ca27d0
```

Comment:
This was an accident, I am going to close and wait until `plugin-generator` is merged and then re do is on top of `plugin-manager`.